src/User32/User32+DPI_AWARENESS_CONTEXT.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <summary>
+    /// Contains nested type <see cref="DPI_AWARENESS_CONTEXT"/>
+    /// </summary>
+    public partial class User32
+    {
+        /// <summary>
+        /// Identifies the awareness context for a window.
+        /// </summary>
+        /// <remarks>DPI_AWARENESS_CONTEXT values should never be compared directly. Instead, use <see cref="AreDpiAwarenessContextsEqual"/></remarks>
+        public class DPI_AWARENESS_CONTEXT : SafeHandle
```

Comment:
Given this handle is _never_ released, I don't think we should use `SafeHandle` as a base type, both because its misleading and because each instance of these types adds load to the CLR as it has to track these for finalization, I believe.
I'd be inclined to just use an enum to define these well known values, if all valid values are well known ones (not really private handles at all). If there are other values without names, then IIRC we typically handle this by typing it as an `IntPtr` or `int` and then define constants in the containing type (e.g. `User32`) for the well-known values.

Reply:
Thanks for chiming in here. 

The gotcha here is that all valid values are NOT well-known ones. The OS returns DPI_AWARENESS_CONTEXT handles that are different that -1, -2, -3, and -4. In fact, I've never seen these values returned by the OS - the handles returned by the OS are always different. So we end up with AreDpiAwarenessContextsEqual(new IntPtr(-1), <some handle returned by OS>) == true frequently. 

That said, I'll make these well known IntPtr values in User32.cs since I don't have a strong preference. These are not enums though - these are handles, so these would be defined as static readonly IntPtr values (alongwith corresponding changes to API's that consume these).

Reply:
Will do - expect another commit today.